### PR TITLE
Implemented dnn-select

### PR DESCRIPTION
**Pull request by valadas**
Implemented dnn-select

### CODE REVIEW

File: packages/stencil-library/src/components/dnn-select/dnn-select.stories.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          ?disable-validity-reporting=${args["disable-validity-reporting"]}
+          ?disabled=${args.disabled}
+          help-text=${ifDefined(args["help-text"])}
+          label=${ifDefined(args.label)}
+          name=${ifDefined(args.name)}
+          ?required=${ifDefined(args.required)}
+          value=${ifDefined(args.value)}
+          @valueChange=${e => eventsFromNames.valueChange(e)}
+        >
+          <option value="">-- Select an option </option>
```
Comment (david-poindexter) [inline packages/stencil-library/src/components/dnn-select/dnn-select.stories.ts:59]:
```suggestion
          <option value="">-- Select an option --</option>
```

**Review by david-poindexter — CHANGES_REQUESTED** [review]

**Review by david-poindexter — APPROVED** [review]

File: packages/stencil-library/src/components/dnn-select/dnn-select.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  /** Fires when the value has changed and the user exits the input. */
+  @Event() valueChange: EventEmitter<string>;
+
+  private slot: HTMLSlotElement;
+  private select: HTMLSelectElement;
+
+  componentWillLoad() {
+    if (this.name === undefined)
+    {
+      this.name = `dnn-input-${Math.floor(Math.random() * 1000000)}`;
```
Comment (david-poindexter) [inline packages/stencil-library/src/components/dnn-select/dnn-select.tsx:46]:
```suggestion
      this.name = `dnn-select-${Math.floor(Math.random() * 1000000)}`;
```

**Review by david-poindexter — APPROVED** [review]

**Comment by valadas** [comment]
Good catches

**Pull request merged**